.github/workflows/stale.yml

```
@@ -0,0 +1,14 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 3 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-pr-message: 'This PR is stale because it has not received activity for more than 7 days. Remove stale label or comment or this will be closed in 7 days.'
+          days-before-stale: 7
```

I'd propose at least doubling this. It's quite usual that something is put on hold after a PR was pushed and the responsible gets involved to something else while waiting for resolution or reviews, etc. In my personal opinion, a couple weeks of waiting is not a big deal.

```suggestion
          days-before-stale: 14
```

---

Meaning a PR would be `alive` for 3 weeks?
Stale means a warning, removing it is OK

I would propose to use `exempt-issue-labels` if needed

---

I thought exactly the same as @guicassolato

---

Right. Alert in 2 weeks, closed in another one if nothing changes.